src/app/[lng]/superfest/[slug]/page.tsx

```diff
@@ -44,10 +44,8 @@ import { notFound } from 'next/navigation';
 
 export default async function Page({ params }: { params: { slug: string } }) {
   const { data, url } = await getQuestBySlug(params.slug);
-
-  if (!data?.data?.[0]) {
+  if (!data) {
```

Comment:
Same as above

Reply:
Yes, I have updated this everywhere where `getQuestBySlug` is being called. Check my answer above.